ARGO/include/Utilities.h

```diff
@@ -66,7 +66,12 @@ namespace Utilities
 	const int AUDIO_CHANNELS_PER_PLAYER = 8;
 	const int AUDIO_PLAYERS_FIRE_CHANNELS = AUDIO_MIX_CHANNELS - AUDIO_CHANNELS_PER_PLAYER * 4;
 
-	static const int S_NUMBER_OF_SCREENS = 7; 
+	static const int S_NUMBER_OF_SCREENS = 7;
+
+	//Flow field values
+	const int MAX_FLOW_FIELD_WEIGHT = 20;
+
+	const float RADIANS_TO_DEGREES = 57.2958;
```

Comment:
glm has  two functions radians() and degrees() that converts between radians and degrees